src/components/PharmacyList/PharmacyList.css

```diff
@@ -0,0 +1,6 @@
+.wrapper {
```

Comment:
Can we use CSS modules here to avoid global css clashing across containers and components.

Reply:
Wait, this shouldn't have been included at all, what the heck